spec/rubocop/cop/lint/to_enum_arguments_spec.rb

```diff
@@ -106,11 +106,10 @@ def m(x, &block)
     RUBY
   end
 
-  it 'registers an offense when enumerator is created for another method' do
-    expect_offense(<<~RUBY)
+  it 'does not register an offense when enumerator is created for another method' do
+    expect_no_offenses(<<~RUBY)
       def m(x)
-        return to_enum(:not_m) unless block_given?
-               ^^^^^^^^^^^^^^^ Ensure you correctly provided all the arguments.
+        return to_enum(:not_m, x)
```

Comment:
But when `return to_enum(:not_m)` (as in the original) is used - this should also not register an offense. But it erroneously generates one.

Reply:
It's a code smell I think. Maybe parameter (default = don't accept)?